<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meteor_radio_main_window</class>
 <widget class="QMainWindow" name="meteor_radio_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1409</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1409</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actOpen"/>
    <addaction name="actConnect_to_DB"/>
    <addaction name="act_Disconnect_from_DB"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Stations">
    <property name="title">
     <string>&amp;Stations</string>
    </property>
    <addaction name="actStatParameters"/>
    <addaction name="actSortResults"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Stations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actStatParameters">
   <property name="text">
    <string>Stations parameters ...</string>
   </property>
  </action>
  <action name="actConnect_to_DB">
   <property name="text">
    <string>Connect to DB ...</string>
   </property>
  </action>
  <action name="act_Disconnect_from_DB">
   <property name="text">
    <string>&amp;Disconnect from DB</string>
   </property>
  </action>
  <action name="actSortResults">
   <property name="text">
    <string>&amp;Sort results</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
